code/integration-tests/runtime-tests/runtime-tests.nix

```diff
@@ -44,9 +44,9 @@
             ((TRIES=TRIES+1))
             sleep 4
           done
-
-          # here nodes are up and running, binaries in path, npm is here too
-
+          npm install
```

Comment:
you should cd into tests directory as they are cloned on CI

Reply:
`cd WHERE_IN_REPO_MY_TESTS`

Reply:
how to cd into cloned CI repo check out GH action docs - it should be specified

Reply:
added cd into test folder

Reply:
same error. i pretty sure binary is here. because when nix runs in folder.it puts files to here may be npm ignores path?